prow/scripts/provision-vm-and-start-kyma.sh

```diff
@@ -24,15 +24,52 @@ else
     LABELS=(--labels "pull-number=$PULL_NUMBER,job-name=kyma-integration")
 fi
 
+POSITIONAL=()
+while [[ $# -gt 0 ]]
+do
+
+    key="$1"
+
+    case ${key} in
+        --image)
+            IMAGE="$2"
```

Comment:
Please add a testing phase here. We should verify if the image exists in gcloud (`gcloud compute images describe`)